src/ui/popup.js

```diff
@@ -70,11 +71,11 @@ class Popup extends Evented {
      */
     addTo(map) {
         this._map = map;
-        this._map.on('move', this._update);
+        this._map.on('move', this._onMove);
         if (this.options.closeOnClick) {
             this._map.on('click', this._onClickClose);
         }
-        this._update();
+        this._update(false);
```

Comment:
Agreed -- I've changed this to always apply the wrapping. Only the logic based on prior position is bypassed on the update immediately following `setLatLng`.